diff --git a/CMakeLists.txt b/CMakeLists.txt
index 15cf8ff..afd700c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -143,7 +143,7 @@ else()
     set(INSTALL_CMAKE     "cmake")                  # ./cmake
     set(INSTALL_EXAMPLES  ".")                      # ./
     set(INSTALL_DATA      ".")                      # ./
-    set(INSTALL_BIN       ".")                      # ./
+    set(INSTALL_BIN       "bin")                    # ./bin
     set(INSTALL_SHARED    "lib")                    # ./lib
     set(INSTALL_LIB       "lib")                    # ./lib
     set(INSTALL_INCLUDE   "include")                # ./include
